docs/source/technical_tutorials/working_with_r/gedi_access_r.ipynb

```diff
@@ -0,0 +1,906 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    # search for the first granules
</pre><p>It is probably out-of-scope here, but connecting this to a real application like "read GEDI for a specific area + time" rather than just the first 10 granules returned would be really cool. I know that working with GEDI data in an efficient way is really hard so maybe the simple case is good enough here since it unblocks a user who just needs to learn how to open the files in R!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/475/discussion/#comment-2021226101'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>It's reasonable that we would refine it to an AOI and Time window. We should add that so people can see how it's done. But you're right the most critical part of this tutorial is actually showing the ability to read a raw HDF5 file in R more than anything else. GEDI just happened to be a common case for MAAP users.</p><p> </p>